### PR TITLE
Patch Byebug C code

### PR DESCRIPTION
**Pull request by bjfish**

### CODE REVIEW

File: lib/cext/preprocess.rb [hunk 1]
```diff
@@ -228,6 +248,40 @@ def read_write_field(struct_var_name, field_name, leaking)
         replacement: MYSQL2_FREE_STATEMENT
       }
     ]
+  },
+  'byebug.c' => {
+    gem: 'byebug',
+    patches: [
+      {
+        match: /\b(static VALUE .*?) = (?:Qfalse|Qnil);/,
```
Comment (eregon) [inline lib/cext/preprocess.rb]: `\w+` would be more precise and avoid potentially matching functions or so.

File: lib/cext/preprocess.rb [hunk 2]
```diff
@@ -228,6 +248,40 @@ def read_write_field(struct_var_name, field_name, leaking)
...
+        replacement: '\1;'
+      },
+      {
+        match: /VALUE threads = Qnil;/,
```
Comment (eregon) [inline lib/cext/preprocess.rb]: Is this at the start of a line? In that case an anchor like `^` would be useful so local variables are less likely to be matched.

File: lib/cext/preprocess.rb (hunk 1 above)
Comment (bjfish) [reply lib/cext/preprocess.rb]: Fixed.

File: lib/cext/preprocess.rb (hunk 2 above)
Comment (bjfish) [reply lib/cext/preprocess.rb]: Yes, fixed.

**Pull request merged**